soccer.js

```diff
@@ -20,7 +20,21 @@ const getPointsFromResult = function getPointsFromResult(result) {
 // including wins, draws, and losses i.e. 'wwdlw'
 // Returns total number of points won
 
-
+function getTotalPoints(string){
```

Comment:
Don't overlook the helpers provided. See how the object `RESULT_VALUES` and function `getPointsFromResult` are here to do some of this work for you?